components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentTypes.java

```diff
@@ -19,15 +19,28 @@
  */
 public class ContentTypes
 {
+  /**
+   * Plain text.
+   */
   public static final String TEXT_PLAIN = "text/plain";
 
+  /**
+   * Generic HTML content type.
+   */
   public static final String TEXT_HTML = "text/html";
 
-  public static final String TEXT_XML = "text/xml";
```

Comment:
text/xml is still a valid content-type, and this class is meant to hold onto constants, not suggest what formats use them for, or enforce any sort of best usage of content-types or anything like that.

I also don't care what Tika has for this, they aren't directly related.